src/chocolatey.resources/helpers/functions/Install-ChocolateyInstallPackage.ps1

```diff
@@ -327,6 +327,18 @@ Pro / Business supports a single, ubiquitous install directory option.
     $env:ChocolateyExitCode = Start-ChocolateyProcessAsAdmin "$msiArgs" "$($env:SystemRoot)\System32\msiexec.exe" -validExitCodes $validExitCodes -workingDirectory $workingDirectory
   }
 
+   if ($fileType -like 'msp') {
+    $msiArgs = "/update `"$fileFullPath`""
+    if ($overrideArguments) {
+      Write-Host "Overriding package arguments with '$additionalInstallArgs' (replacing '$silentArgs')";
+      $msiArgs = "$msiArgs $additionalInstallArgs";
+    } else {
+      $msiArgs = "$msiArgs $silentArgs $additionalInstallArgs";
+    }
+
+    $env:ChocolateyExitCode = Start-ChocolateyProcessAsAdmin "$msiArgs" "$($env:SystemRoot)\System32\msiexec.exe" -validExitCodes $validExitCodes -workingDirectory $workingDirectory

```

Comment:
```suggestion
    $env:ChocolateyExitCode = Start-ChocolateyProcessAsAdmin $msiArgs "$($env:SystemRoot)\System32\msiexec.exe" -validExitCodes $validExitCodes -workingDirectory $workingDirectory
```

I'd probably recommend using named arguments for everything as well, and fixing the casing on those ones already there.